model/models.go

```diff
@@ -154,7 +154,7 @@ type Resource struct {
 	Privacy         string     `sql:",notnull" json:",omitempty"`
 	Type            string     `sql:",notnull" json:",omitempty"`
 	Views           int64      `json:",omitempty"`
-	Recommendations int64      `json:",omitempty"`
+	Recommendations int64      `sql:",notnull" json:",omitempty"`
```

Comment:
Can you test the create resource endpoint please. I'm thinking this might be a breaking change for it.

Reply:
Are you saying write test for this or run the tests we already have for it ?

Reply:
@mikey2020 I don't remember what the implementation of the `create resource` endpoint looks like. But I'm thinking a not null directive could have broken something. So in the mean time you could just run all the tests we already have. I will take a look at the implementation tonight. You can take a look at it too so we can be sure nothing breaks.